nautobot_firewall_models/models/capirca_models.py

```diff
@@ -0,0 +1,65 @@
+"""Models for the Capirca Configurations."""
```

Comment:
This was the pattern set by @whitej6 so i'll defer to him.

Reply:
Was just trying to follow pattern

Reply:
Core = models exposed to user
Through = Indirect models for permissions or overloading the M2M relationship to allow for additional attrs. 

I am not opposed to leaving as is and it may make sense to spit out the models to their own domains similar to the nav menu.